LR_Tourist/DI/DI.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Remove this project. It's useless project

Reply:
fixed